common/src/autogluon/common/utils/try_import.py

```diff
@@ -30,7 +30,7 @@ def try_import_mxboard():
 
 
 def try_import_ray() -> ModuleType:
-    RAY_MAX_VERSION = "2.11.0"  # sync with core/setup.py
+    RAY_MAX_VERSION = "2.40.0"  # sync with core/setup.py
```

Comment:
`tabular_medium` in AG-Bench I assume means `medium_quality`. `medium_quality` does not use Ray. We need to test `best_quality` which uses Ray.

Reply:
That's good to know. Let me run a benchmark with `best_quality`